src/components/PurchaseOrder/CloseOrder/CloseOrderModal.js

```diff
@@ -92,11 +92,13 @@ class CloseOrderModal extends Component {
           <Row>
             <Col xs={12}>
               <Select
-                dataOptions={reasons}
+                autoFocus
                 label={<FormattedMessage id="ui-orders.closeOrderModal.reason" />}
                 onChange={e => this.setState({ reason: e.target.value })}
-                required
               >
+                <FormattedMessage id="ui-orders.dropdown.select">
+                  {(message) => <option value="" disabled selected hidden>{message}</option>}
```

Comment:
If the reason is not required field, maybe it makes sense just to add `placeholder=" "` property

Reply:
added placeholder